### PR TITLE
docs: add Playground section to "Getting Started"

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR adds the Playground to the Getting Started section in the docs. The idea is to put forward the Playground as the first place for people to go and learn about Gno, instead of having to set up a local chain and all of the things that follow.

I suggest reviewing this PR by running a local version of the docs, with `yarn start` in the `misc/docusaurus` folder. (when deployment preview @albttx)

This is one of 2 PRs that will update the Getting started section in the Docs. The next PR will update the Local setup section to put `gnodev` as the first option to get started. 

Here is the latest version of the PR: https://www.loom.com/share/636afb0d48ed4bf7ac8043b17463b8fd?sid=0c055f69-a5a8-4ddf-8d42-f550db17dbe3

<details><summary>Contributors' checklist...</summary>

- [x] Added new tests, or not needed, or not feasible
- [x] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [x] Updated the official documentation or not needed
- [x] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [x] Added references to related issues and PRs
- [x] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: docs/getting-started/playground/start.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+like to deploy to, such as [Portal Loop](../../concepts/portal-loop.md) or local,
+and click deploy.
+
+:::info
+Even if you don't have testnet tokens, the Playground will automatically provide
+you with enough to cover the gas cost at the time of deployment.
+:::
+
+### Format
+Format will format your code, using `gofmt`.
```
Comment (salmad3) [inline docs/getting-started/playground/start.md:65]: Should we provide more context? Something like 
```
The 'Format' feature utilizes the Monaco editor and [`gofmt`](https://pkg.go.dev/cmd/gofmt) to automatically refine and standardize your Gno code's syntax.
```

**Comment by ilgooz** [comment]
@salmad3 can you please review?

File: docs/getting-started/playground/start.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+explained to newcomers. For reading more about Gno.land, 
+check out the [Concepts](../../concepts/concepts.md) section.
+
+To get started writing Gno code, check out the
+[How-to](../../how-to-guides/how-to-guides.md) section, the `examples/` folder on
+the [Gno monorepo](https://github.com/gnolang/gno), or one of many community projects and tutorials found in the 
+[awesome-gno](https://github.com/gnolang/awesome-gno/blob/main/README.md) repo on GitHub.
+
+
+
```
Comment (salmad3) [inline docs/getting-started/playground/start.md:105]: nit: extra newlines

File: docs/getting-started/playground/start.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+The Playground has the following features:
+- `Share` - Get a permanent, shareable link to the code currently found in the playground
+- `Deploy` - Connect your wallet and deploy your code to a Gno.land network
+- `Format` - Format your Gno code
+- `Run` - Run a specific expression on your code
+- `Test` - Run tests on your code
+- `REPL` - Experiment and troubleshoot in real-time using the GnoVM with
+interactive REPL features (experimental)
+
+Let's dive into each of the Playground features.
```
Comment (salmad3) [inline docs/getting-started/playground/start.md:40]: It seems we dive into some more than others.

Some are a bit redundant with the descriptions in the list above, so it seems we could provide more context (like with REPL)

Comment (salmad3) [reply docs/getting-started/playground/start.md:40]: We can also consider starting each of these subsections consistently, such as "The <feature-name> feature ..."

**Review by salmad3 — CHANGES_REQUESTED** [review]
Thanks for this @leohhhn, Left some suggestions.

Also, should the links use absolute paths instead of relative?

File: docs/getting-started/playground/start.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
+---
+id: start
+---
+
+# Gno Playground
```
[the remark this answers is not in the document]
Comment (leohhhn) [reply docs/getting-started/playground/start.md:5]: I would keep it simple tbh, as to not create more confusion later when the IDE is introduced

File: docs/getting-started/playground/start.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
+## Prerequisites
+
+- **Internet connection**
+- **A keypair in a Gno.land wallet, such as [Adena](https://adena.app)**
+
+To fully utilize Playground features, you will need to install a Gno.land web
+browser wallet, such as [Adena](https://www.adena.app/), and create a keypair. This will allow you to
```
[the remark this answers is not in the document]
Comment (leohhhn) [reply docs/getting-started/playground/start.md:21]: Can this work? [8a007c3](https://github.com/gnolang/gno/pull/1773/commits/8a007c395fd02703b34c84f7a013a63c3f4de845)

File: docs/getting-started/playground/start.md (hunk 1 above)
Comment (leohhhn) [reply docs/getting-started/playground/start.md]: I was hesitant to include mentioning "Monaco" since it doesn't seem to me that a lot of people know about it, but in this case more info is good imo. [b2cb449](https://github.com/gnolang/gno/pull/1773/commits/b2cb4499caf700d588306ac892a5d3cddbb1abd0)

File: docs/getting-started/playground/start.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (leohhhn) [reply docs/getting-started/playground/start.md]: Actually, just removed that whole paragraph. [2f9f02f](https://github.com/gnolang/gno/pull/1773/commits/2f9f02f4ecd44e9cd05dd840fbbc1ea339274be8)

File: docs/getting-started/playground/start.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+and deploying simple with its diverse set of tools and features. Users can
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
+## Prerequisites
+
+- **Internet connection**
+- **A keypair in a Gno.land wallet, such as [Adena](https://adena.app)**
+
+To fully utilize Playground features, you will need to install a Gno.land web
```
[the remark this answers is not in the document]
Comment (leohhhn) [reply docs/getting-started/playground/start.md]: [2f9f02f](https://github.com/gnolang/gno/pull/1773/commits/2f9f02f4ecd44e9cd05dd840fbbc1ea339274be8)

File: docs/getting-started/playground/start.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+- `Format` - Format your Gno code
+- `Run` - Run a specific expression on your code
+- `Test` - Run tests on your code
+- `REPL` - Experiment and troubleshoot in real-time using the GnoVM with
+interactive REPL features (experimental)
+
+Let's dive into each of the Playground features.
+
+### Share
+
```
[the remark this answers is not in the document]
Comment (leohhhn) [reply docs/getting-started/playground/start.md:41]: [60e5d31](https://github.com/gnolang/gno/pull/1773/commits/60e5d31115b5a800dc0d4896ad104b8929a68a3b)

File: docs/getting-started/playground/start.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+This feature allows users to get a permanent shortlink to the code found in the
+playground at the time of clicking. This way, Gno code can be shared easily. 
+
+### Deploy
+
+Deploy allows users to quickly deploy their Gno code to the chain. After connecting 
+a Gno.land wallet, users will be able to choose which package path to deploy to,
+as well as which network.
```
[the remark this answers is not in the document]
Comment (leohhhn) [reply docs/getting-started/playground/start.md]: [2315ebd](https://github.com/gnolang/gno/pull/1773/commits/2315ebd891d0095feb0a90e5fb1234cc0d9d770c)

File: docs/getting-started/playground/start.md (hunk 2 above)
Comment (leohhhn) [reply docs/getting-started/playground/start.md]: [5459e48](https://github.com/gnolang/gno/pull/1773/commits/5459e481e17b9ee640678361257516327d00cd14)

**Comment by leohhhn** [comment]
Hey @salmad3, thanks for the review. I fixed all of the things you mentioned, except the ones that are waiting for information from @ilgooz.

About absolute paths, Manfred requested to keep them relative: https://github.com/gnolang/gno/pull/1357

File: docs/getting-started/playground/start.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+[here](../../concepts/gno-test.md).
+
+### REPL (experimental)
+
+The **REPL** Feature, although experimental, will let you experiment with the GnoVM
+in REPL mode. 
+
+## Learning about Gno.land & writing Gno code
+
+Gno.land is a complex technical system, and as such many concepts need to be
```
Comment (salmad3) [inline docs/getting-started/playground/start.md:97]: While true, how about something more welcoming? Something like:

"If you're new here, don't worry—content is regularly produced to breakdown Gno.land to explain its features. Dive into the essentials of Gno.land by exploring the Concepts section."

File: docs/getting-started/playground/start.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+The **Test** feature will look for `_test.gno` files in your playground and run 
+the`gno test -v` command on them. Testing your code will open a terminal that
+will show you the output of the test. Read more about how Gno tests work
+[here](../../concepts/gno-test.md).
+
+### REPL (experimental)
+
+The **REPL** Feature, although experimental, will let you experiment with the GnoVM
+in REPL mode.
```
Comment (salmad3) [inline docs/getting-started/playground/start.md:93]:
```suggestion
It provides a command-line interface for hands-on learning, iterative testing, and swift prototyping.
```

File: docs/getting-started/playground/start.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+# Gno Playground
+
+## Overview
+
+The Gno Playground is a web editor and sandbox that enables developers to 
+interactively work with the Gno language. It makes coding, testing,
+and deploying simple with its diverse set of tools and features. Users can
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
```
Comment (salmad3) [inline docs/getting-started/playground/start.md:14]: Not a strong suggestion: we can also call out its similarity to [go playground](https://go.dev/play/)

**Review by salmad3 — CHANGES_REQUESTED** [review]

**Comment by zivkovicmilos** [comment]
@leohhhn 

Can you please include a Loom / screenshots in the PR description?

It's very cumbersome to clone your fork, swap over to this PR branch, and run the UI locally

File: docs/getting-started/playground-start.md [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+
+The Gno Playground is an innovative web-based editor and sandbox that enables developers to 
+interactively work with the Gno language. It makes coding, testing,
+and deploying simple with its diverse set of tools and features. Users can
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
+## Prerequisites
+
+- **Internet connection**
```
Comment (zivkovicmilos) [inline docs/getting-started/playground-start.md:17]: Is this really necessary?

File: docs/getting-started/playground-start.md [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+- **A Gno.land compatible wallet**
+  - Currently, [Adena](https://www.adena.app/) is the go-to wallet for
```
Comment (zivkovicmilos) [inline docs/getting-started/playground-start.md:19]: I would rephrase this as `...is the preferred wallet...`

File: docs/getting-started/playground-start.md [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+To get started, visit the Playground at [play.gno.land](https://play.gno.land). You will be greeted with a
+simple `package.gno` file:
+
+![default_playground](../assets/getting-started/playground/default_playground.png)
+
+The Playground has the following features:
+- `Share` - Generate a unique, short, and shareable identifier for your Gno code.
+- `Deploy` - Connect your wallet and publish your code on Gno.land.
+- `Format` - Automatically adjust your Gno code's structure and style for optimal readability and consistency.
+- `Run` - Execute a particular expression within your code to validate its functionality and output.
```
Comment (zivkovicmilos) [inline docs/getting-started/playground-start.md:33]: What do you think about expanding a bit more on this functionality? Specifically on what `expression` means

File: docs/getting-started/playground-start.md [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+
+[![run_example](../assets/getting-started/playground/run.png)](https://play.gno.land/p/nBq2W8drjMy)
+
+Running `println(Render("Gnopher"))` will display the following output:
+
+```bash
+Hello Gnopher!
+```
+
+View the code [here](https://play.gno.land/p/nBq2W8drjMy).
```
Comment (zivkovicmilos) [inline docs/getting-started/playground-start.md:82]: Why is there no mention in this section that you can run actual functions?

**Review by zivkovicmilos — CHANGES_REQUESTED** [review]
Please see my comment about including a Loom / screenshot so these changes are clear 🙏 

I've left a few minor comments

File: docs/getting-started/playground-start.md (hunk 12 above)
Comment (leohhhn) [reply docs/getting-started/playground-start.md]: It was a leftover, removed it: [df268e5](https://github.com/gnolang/gno/pull/1773/commits/df268e52a3cc9bd2b0f80fb00acf940c299df150)

File: docs/getting-started/playground-start.md (hunk 13 above)
Comment (leohhhn) [reply docs/getting-started/playground-start.md]: [0051394](https://github.com/gnolang/gno/pull/1773/commits/0051394bd6090098946bbfccaa0b67db0a9f7048)

File: docs/getting-started/playground-start.md (hunk 15 above)
Comment (leohhhn) [reply docs/getting-started/playground-start.md:81]: Because running a normal function that does not have a `print` method inside, or just not printing the output of a function, will just give you an empty terminal. This feature is in discussions to be improved with DevX.

File: docs/getting-started/playground-start.md (hunk 14 above)
Comment (leohhhn) [reply docs/getting-started/playground-start.md:32]: I would like to keep this specific item list relatively short, and expand on it in the individual feature sections.

**Comment by leohhhn** [comment]
@zivkovicmilos 

Here is the latest version of the PR: https://www.loom.com/share/636afb0d48ed4bf7ac8043b17463b8fd?sid=0c055f69-a5a8-4ddf-8d42-f550db17dbe3

Hopefully with the docs migration we get a way to preview PRs easily.

File: docs/getting-started/playground-start.md (hunk 15 above)
Comment (zivkovicmilos) [reply docs/getting-started/playground-start.md:81]: You can get the return value by running `Run` on the Playground UI, so I'm not sure what the confusion is about

Comment (leohhhn) [reply docs/getting-started/playground-start.md:81]: Here's what I'm talking about: https://www.loom.com/share/3ee22ee79ee24677935703ba33f42c5d?sid=52c89403-078e-4c96-bed7-2ac779a2f801

Comment (zivkovicmilos) [reply docs/getting-started/playground-start.md:81]: If I'm not going crazy, calling `Run` would give you the output of the method:
```shell
("Hello" string)
```

**Review by zivkovicmilos — APPROVED** [review]
Thank you for resolving the convos 🙏

File: docs/getting-started/playground-start.md (hunk 15 above)
Comment (leohhhn) [reply docs/getting-started/playground-start.md:81]: Linked issue for reference: https://github.com/gnostudio/studio/issues/581
